common/src/main/java/org/broadleafcommerce/common/rule/MvelHelper.java

```diff
@@ -80,7 +84,11 @@ public static Object convertField(String type, String fieldValue) {
             if (type.equals(SupportedFieldType.BOOLEAN.toString())) {
                 return Boolean.parseBoolean(fieldValue);
             } else if (type.equals(SupportedFieldType.DATE.toString())) {
-                return FormatUtil.getTimeZoneFormat().parse(fieldValue);
+                try {
```

Comment:
Change to use a length check instead of a  ... getTimeZoneFormat().length() instead of relying on an exception.    Need to try and remove and recreate and then add the length check.